lib/docker/executeDockerBuild.ts

```diff
@@ -123,22 +148,19 @@ export function executeDockerBuild(options: DockerOptions): ExecuteGoal {
                 break;
         }
 
+        // Determine image name, registry urls
         const imageName = await optsToUse.dockerImageNameCreator(project, goalEvent, optsToUse, context);
-        const images = imageName.tags.map(tag => `${imageName.registry ? `${imageName.registry}/` : ""}${imageName.name}:${tag}`);
```

Comment:
Put this back with the above change to have the image name creator return all fully qualified image names.

Reply:
Will update; as discussed.